sample_apps/common/example_app.cc

```diff
@@ -283,28 +288,70 @@ int main(int an, char **av) {
   an = 1;
   ::testing::InitGoogleTest(&an, av);
 
+  // clang-format off
   if (FLAGS_operation == "") {
-    printf("%s --print_all=true|false --operation=op "
-           "--policy_host=policy-host-address "
-           "--policy_port=policy-host-port\n",
-           av[0]);
-    printf("\t --data_dir=-directory-for-app-data "
-           "--server_app_host=my-server-host-address "
-           "--server_app_port=server-host-port\n");
-    printf("\t --policy_cert_file=self-signed-policy-cert-file-name "
-           "--policy_store_file=policy-store-file-name\n");
+    printf("                                                                            (Defaults)\n");
+    printf("%s --operation=<op>                                        ; %s", av[0], "(See below)");
+    printf("\n\
+                  --policy_host=policy-host-address                       ; %s\n\
+                  --policy_port=policy-host-port                          ; %d\n\
+                  --server_app_host=my-server-host-address                ; %s\n\
+                  --server_app_port=my-server-port-number                 ; %d\n\
+                  --data_dir=-directory-for-app-data                      ; %s\n\
+                  --policy_cert_file=self-signed-policy-cert-file-name    ; \n\
+                  --policy_store_file=policy-store-file-name              ; %s\n\
+                  --print_all=true|false",
+                  FLAGS_policy_host.c_str(),
+                  FLAGS_policy_port,
+                  FLAGS_server_app_host.c_str(),
+                  FLAGS_server_app_port,
+                  FLAGS_data_dir.c_str(),
+                  FLAGS_policy_store_file.c_str());
+#ifdef SIMPLE_APP
+    printf("\n\
+                  --platform_file_name=platform-cert-bin-file-name        ; %s\n\
+                  --platform_attest_endorsement=endorsement-bin-file-name ; %s\n\
+                  --measurement_file=measurement-bin-file-name            ; %s\n\
+                  --attest_key_file=attest-key-bin-file-name              ; %s\n",
+                  FLAGS_platform_file_name.c_str(),
+                  FLAGS_platform_attest_endorsement.c_str(),
+                  FLAGS_measurement_file.c_str(),
+                  FLAGS_attest_key_file.c_str());
```

Comment:
Above block of diffs improves the output from `--help` to be a bit more readable.

Reply:
Output now looks like this:

```
cc-sdb-vm:[237] $ ./example_app.exe
                                                                            (Defaults)
./example_app.exe --operation=<op>                                        ; (See below)
                  --policy_host=policy-host-address                       ; localhost
                  --policy_port=policy-host-port                          ; 8123
                  --server_app_host=my-server-host-address                ; localhost
                  --server_app_port=my-server-port-number                 ; 8124
                  --data_dir=-directory-for-app-data                      ; ./app1_data/
                  --policy_cert_file=self-signed-policy-cert-file-name    ;
                  --policy_store_file=policy-store-file-name              ; store.bin
                  --print_all=true|false
                  --platform_file_name=platform-cert-bin-file-name        ; platform_file.bin
                  --platform_attest_endorsement=endorsement-bin-file-name ; platform_attest_endorsement.bin
                  --measurement_file=measurement-bin-file-name            ; example_app.measurement
                  --attest_key_file=attest-key-bin-file-name              ; attest_key_file.bin


Operations are: cold-init, get-certified, run-app-as-client, run-app-as-server
```